### PR TITLE
Add missing error handling for when the platform version is empty.

### PR DESCRIPTION
**Pull request by adrian-prantl**
rdar://100570330

### CODE REVIEW

**Comment by adrian-prantl** [comment]
@swift-ci test

**Comment by adrian-prantl** [comment]
@swift-ci test

**Review by augusto2112 — APPROVED** [review]

**Comment by adrian-prantl** [comment]
@swift-ci test

File: lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp [hunk 1]
```diff
@@ -1997,25 +1997,38 @@ lldb::TypeSystemSP SwiftASTContext::CreateInstance(
...
           os << target_triple.getArchName() << '-';
           os << target_triple.getVendorName() << '-';
           os << llvm::Triple::getOSTypeName(target_triple.getOS());
-          os << version.getAsString();
+          os << platform_version.getAsString();
         }
         computed_triple = llvm::Triple(buffer);
       } else {
+        LOG_PRINTF(GetLog(LLDBLog::Types),
+                   "Completing triple based on main binary load commands.");
```
Comment (kastiglione) [inline lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp:2031]: Nice!

**Review by kastiglione — COMMENTED** [review]
should this function have one last log that reports the resulting triple?

File: lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp (hunk 1 above)
Comment (adrian-prantl) [reply lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp:2031]: > should this function have one last log that reports the resulting triple?

SetTriple(), which is called afterwards also logs.

**Pull request merged**